app/stac_api/serializers.py

```diff
@@ -416,6 +491,27 @@ class Meta:
             'updated'
         ]
 
+    # NOTE: when explicitely declaring fields, we need to had the validation as for the field
+    # in model !
+    type = serializers.CharField(source='media_type', max_length=200)
+    # Here we need to explicitely define these fields with the source, because they are renamed
+    # in the get_fields() method
+    eo_gsd = serializers.FloatField(source='eo_gsd', required=False, allow_null=True)
+    geoadmin_lang = serializers.ChoiceField(
+        source='geoadmin_lang', choices=['de', 'fr', 'it', 'rm', 'en']
```

Comment:
probably an `allow_blank = True` is missing here, as in the Language class "NONE" (i.e. "") is a valid option.

Reply:
Good point, I'll add the `required=False` and `allow_blank=True` to match the SPEC and add a comment in https://jira.swisstopo.ch/browse/BGDIINF_SB-1372 which will fully implement the asset deserializer